op-conductor/health/monitor_test.go

```diff
@@ -105,21 +105,23 @@ func (s *HealthMonitorTestSuite) TestUnhealthyUnsafeHeadNotProgressing() {
 
 	rc := &testutils.MockRollupClient{}
 	ss1 := mockSyncStatus(now, 5, now-8, 1)
-	for i := 0; i < 5; i++ {
+	unsafeBlocksInterval := 10
+	for i := 0; i < unsafeBlocksInterval+2; i++ {
```

Comment:
The reason for the extra tick past unsafeBlocksInterval + 1 is just that the "unsafe head is falling behind the unsafe interval" log message isn't displayed until the last tick and it's nice to see that when running locally.